src/interface.rs

```diff
@@ -9,6 +9,7 @@ use crate::js_doc::JsDoc;
 use crate::node::DeclarationKind;
 use crate::params::ts_fn_param_to_param_def;
 use crate::swc_util::get_location;
+use crate::swc_util::is_false;
```

Comment:
No big deal, but this isn't really related to swc.

Reply:
There was no other good place to put it and I wasn't going to just create another module for it.